modules/ROOT/pages/cache-scope-invalidate.adoc

```diff
@@ -0,0 +1,77 @@
+= Invalidate Cache Entries
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can invalidate a cache key or a complete cache by using the Invalidate Key and Invalidate Cache components respectively.
+Invalidate cache entries when you want the Cache Scope to execute all processors to generate a response, instead of returning a cached response for the existing entry.
```

Comment:
i would rather express this as `want the Cache Scope to recalculate the value for a given cache entry`